src/Chapter1.hs

```diff
@@ -614,9 +628,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int 
+sumLast2 n = tensNumber (abs n) + unitsNumber (abs n)
+    where 
+        tensNumber :: Int -> Int
+        tensNumber x = div (mod x 100) 10
+        unitsNumber :: Int -> Int
+        unitsNumber x = mod x 10
```

Comment:
This is perfect! 👌🏼 

You can notice that `unitsNumber` in this solution is actually the same as `lastDigit` from the previous task!